modules/apigee_edge_teams/src/DefaultTeamPermissionsProvider.php

```diff
@@ -74,6 +74,10 @@ public function permissions(): array {
           'update' => $this->t('Edit any Team Apps'),
           'delete' => $this->t('Delete any Team Apps'),
           'analytics' => $this->t('View analytics of any Team Apps'),
+          'add_api_key' => $this->t('Add API key to any Team Apps'),
+          'revoke_api_key' => $this->t('Revoke API key from any Team Apps'),
+          'delete_api_key' => $this->t('Delete API key from any Team Apps'),
+          'edit_api_products' => $this->t('Edit API products for any Team Apps'),
```

Comment:
```suggestion
          'add_api_key' => $this->t('Add API key to Team Apps'),
          'revoke_api_key' => $this->t('Revoke API key from Team Apps'),
          'delete_api_key' => $this->t('Delete API key from Team Apps'),
          'edit_api_products' => $this->t('Edit API products for Team Apps'),
```
I feel the "any" word changes the meaning as if to say user with permission can manage the api keys regardless of team membership.